apps/dapp-console/app/settings/components/WalletVerificationMethods.tsx

```diff
@@ -0,0 +1,108 @@
+import Image from 'next/image'
+
+import { Text } from '@eth-optimism/ui-components/src/components/ui/text'
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+  Button,
+  Separator,
+} from '@eth-optimism/ui-components'
+
+type VerificationMethod = {
+  id: string
+  icon: string
+  title: string
+  description?: string
+}
+
+const verificationMethods: VerificationMethod[] = [
```

Comment:
could move this to constants later

Reply:
yea if we ever need this in more than one place we can move it out to a constants file